roles/dosomething.sixpack/templates/sixpack.yaml.j2

```diff
@@ -0,0 +1,14 @@
+redis_port: 6379                            # Redis port
+redis_host: localhost                       # Redis host
+redis_prefix: sixpack                       # all Redis keys will be prefixed with this
+redis_db: 15                                # DB number in redis
```

Comment:
For my knowledge, does using a non-default Redis db avoid issues, or is it a personal preference?